Regression: vcc linker options are ignored

If you add to `vcc.options.linker` options like `/Zi` to make debug build it is now ignored.
You get at link time warnings instead: 
```
LINK : warning LNK4044: unrecognized option '/MD'; ignored
LINK : warning LNK4044: unrecognized option '/Zi'; ignored
```
Regression is caused by https://github.com/nim-lang/Nim/commit/d27bc03b21ea066134a297e61d0d18c901300ccc

--- a/compiler/extccomp.nim
+++ b/compiler/extccomp.nim
@@ -162,7 +162,7 @@ compiler vcc:
     buildDll: " /LD",
     buildLib: "lib /OUT:$libfile $objfiles",
     linkerExe: "cl",
-    linkTmpl: "$builddll$vccplatform /Fe$exefile $objfiles $buildgui /link /nologo $options",
+    linkTmpl: "$builddll$vccplatform /Fe$exefile $objfiles $buildgui /nologo $options",
     includeCmd: " /I",
     linkDirCmd: " /LIBPATH:",
     linkLibCmd: " $1.lib",

--- a/config/nim.cfg
+++ b/config/nim.cfg
@@ -320,13 +320,13 @@ tcc.options.always = "-w"
   @if lto_incremental:
    vcc.options.always%= "${vcc.options.always} /GL /Gw /Gy"
    vcc.cpp.options.always%= "${vcc.cpp.options.always} /GL /Gw /Gy"
-   vcc.options.linker %= "${vcc.options.linker} /LTCG:incremental"
-   vcc.cpp.options.linker %= "${vcc.cpp.options.linker} /LTCG:incremental"
+   vcc.options.linker %= "${vcc.options.linker} /link /LTCG:incremental"
+   vcc.cpp.options.linker %= "${vcc.cpp.options.linker} /link /LTCG:incremental"
   @else:
    vcc.options.always%= "${vcc.options.always} /GL"
    vcc.cpp.options.always%= "${vcc.cpp.options.always} /GL"
-   vcc.options.linker %= "${vcc.options.linker} /LTCG"
-   vcc.cpp.options.linker %= "${vcc.cpp.options.linker} /LTCG"
+   vcc.options.linker %= "${vcc.options.linker} /link /LTCG"
+   vcc.cpp.options.linker %= "${vcc.cpp.options.linker} /link /LTCG"
   @end
   clang_cl.options.always%= "${clang_cl.options.always} -flto"
   clang_cl.cpp.options.always%= "${clang.cpp.options.always} -flto"

